dask_jobqueue/core.py

```diff
@@ -548,5 +548,10 @@ def _job_id_from_submit_output(self, out):
 
         return job_id
 
-    def worker_key(self, worker_state):
+    @staticmethod
+    def worker_key(worker_state):
         return _job_id_from_worker_name(worker_state.name)
+
+    @property
+    def scheduler_comm(self):
+        return self.local_cluster.scheduler_comm
```

Comment:
This is for compatibility with `SpecCluster`, which some elements, like adaptive, are starting to expect